src/puppet-web/wechaty-bro.js

```diff
@@ -76,6 +76,7 @@
     glueToAngular()
     connectSocket()
     hookEvents()
+    hookProcess()
```

Comment:
I use `hookProcess` because I think in the future may add other hooks. But for now, the term is not clear, I will modify it.